src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+} from 'react-router-dom';
+import './styles/App.css';
+import Main from './pages/Main';
+
+function App() {
```

Comment:
- [ ] I couldn't find a way to add or delete doctors.  Since you are a team of 4, these are required features.  Kindly implement those so we can meet these requirements.  Everything else is working fine so you are almost there!💪